codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Settings.java

```diff
@@ -176,7 +183,7 @@ public Http2Settings copyFrom(Http2Settings settings) {
         return this;
     }
 
-    Integer getIntValue(int key) {
+    public Integer getIntValue(int key) {
```

Comment:
@nmittler - Correct me if I'm wrong but I'll take a shot...

@normanmaurer - I think because before this change we only allowed access to the "standard" set of settings (only 6 of them) via the other public methods in this class (i.e. `maxHeaderListSize`).

@nmittler - We should consider the duplication between this and `Long get(int)`.  The [spec](https://tools.ietf.org/html/draft-ietf-httpbis-http2-17#section-6.5.1) calls for `value` being `an unsigned 32-bit value.`....we should call this out in the comments if we decide to expose this method publicly.  We may not want to expose this and just leave `Long get(int)` as the public method. WDYT?